<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translate</class>
 <widget class="QDialog" name="Translate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1624</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>BdiDu翻訳</string>
   </property>
   <widget class="QTextEdit" name="translationText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="translationBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="baiduTranslationCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>16</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>BaiDu反向翻訳</string>
   </property>
   <widget class="QTextEdit" name="transToOriginalText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="baiduTranslationReCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Google翻訳</string>
   </property>
   <widget class="QTextEdit" name="translationText_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="translationBox_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="googleTranslationCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>530</y>
     <width>381</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Google反向翻訳</string>
   </property>
   <widget class="QTextEdit" name="transToOriginalText_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="googleTranslationReCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>原文</string>
   </property>
   <widget class="QComboBox" name="originalBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="originalText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>751</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>検索</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnJTOC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>日中</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>最小化</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>クリア</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnCTOJ">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>中日</string>
    </property>
    <property name="shortcut">
     <string>Alt+4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnJTOE">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>日英</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnETOJ">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>英日</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnCTOE">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>中英</string>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtnETOC">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>英中</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>20</y>
     <width>381</width>
     <height>741</height>
    </rect>
   </property>
   <property name="title">
    <string>翻译履历</string>
   </property>
   <widget class="QTextEdit" name="resume">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="resumeBaidu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>百度翻译</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resumeGoogle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Google翻译</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="savefile">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>保存到文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>381</width>
     <height>741</height>
    </rect>
   </property>
   <property name="title">
    <string>分词</string>
   </property>
   <widget class="QTextEdit" name="tanGoTran">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>341</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tanGoWaKe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>originalText</tabstop>
  <tabstop>queryBtnJTOC</tabstop>
  <tabstop>queryBtnCTOJ</tabstop>
  <tabstop>queryBtnJTOE</tabstop>
  <tabstop>queryBtnETOJ</tabstop>
  <tabstop>queryBtnCTOE</tabstop>
  <tabstop>queryBtnETOC</tabstop>
  <tabstop>minBtn</tabstop>
  <tabstop>queryBtn</tabstop>
  <tabstop>originalBox</tabstop>
  <tabstop>translationText</tabstop>
  <tabstop>googleTranslationReCheck</tabstop>
  <tabstop>transToOriginalText_2</tabstop>
  <tabstop>translationBox</tabstop>
  <tabstop>googleTranslationCheck</tabstop>
  <tabstop>clearBtn</tabstop>
  <tabstop>baiduTranslationCheck</tabstop>
  <tabstop>transToOriginalText</tabstop>
  <tabstop>baiduTranslationReCheck</tabstop>
  <tabstop>translationText_2</tabstop>
  <tabstop>translationBox_2</tabstop>
  <tabstop>resume</tabstop>
  <tabstop>resumeBaidu</tabstop>
  <tabstop>resumeGoogle</tabstop>
  <tabstop>savefile</tabstop>
  <tabstop>tanGoTran</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>queryBtn</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearBtn</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnJTOC</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnCTOJ</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnJTOE</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnCTOE</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnETOC</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryBtnETOJ</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
